Makefile.work

```diff
@@ -98,8 +107,14 @@ endif
 
 ifeq ($(CONFIGURED_ARCH),amd64)
 SLAVE_BASE_IMAGE = $(SLAVE_DIR)
+EMULATED_COMPILE = n
```

Comment:
i am not sure how well known this emulated_compile term is. can we rename to MULTIARCH_QEMU_ENVIRON = y? It is perhaps more precise and better known.

Reply:
Obviously, MULTIARCH_QEMU_ENVIRON is more precise and better known, I will rename it.